proxy/proxy.go

```diff
@@ -22,9 +22,10 @@ const (
 	// An example entry would look like this:
 	// 2019-11-09 04:07:55.072 [INF] PRXY: 66.249.69.89 - -
 	// "GET /availability/v1/btc.json HTTP/1.1" "" "Mozilla/5.0 ..."
-	formatPattern  = "- - \"%s %s %s\" \"%s\" \"%s\""
-	hdrContentType = "Content-Type"
-	hdrTypeGrpc    = "application/grpc"
+	formatPattern       = "- - \"%s %s %s\" \"%s\" \"%s\""
+	hdrContentType      = "Content-Type"
+	hdrTypeGrpc         = "application/grpc"
+	defaultServicePrice = 1
```

Comment:
This can then be moved elsewhere, either `aperture.go` where we parse the config, or in `proxy/service.go`.

Reply:
Could you also add a comment noting the units of the constant?